components/camera/camera.go

```diff
@@ -195,52 +167,6 @@ func (vs *sourceBasedCamera) Unsubscribe(ctx context.Context, id rtppassthrough.
 	return errors.New("Unsubscribe unimplemented")
 }
 
-// NewVideoSourceFromReader creates a VideoSource either with or without a projector. The stream type
-// argument is for detecting whether or not the resulting camera supports return
-// of pointcloud data in the absence of an implemented NextPointCloud function.
-// If this is unknown or not applicable, a value of camera.Unspecified stream can be supplied.
-func NewVideoSourceFromReader(
-	ctx context.Context,
-	reader gostream.VideoReader,
-	syst *transform.PinholeCameraModel, imageType ImageType,
-) (VideoSource, error) {
-	if reader == nil {
```

Comment:
@nicksanford, These deletions stem from moving the code to `videosourcewrapper.go` because this file was too confusing for me to understand the private, public, and wrapped video sources.